05-Time-Series-Raster.Rmd

```diff
@@ -202,64 +220,74 @@ xres(NDVI_stack)
 
 ```
 
-> ##Challenge
-> 
-> Before you go any further, answer the following questions about our `RasterStack`.
-> 
-> 1. What is the `CRS`?
-> 2. What is the `resolution` of the data? And what `units` is that resolution in?
+#Challenge: Raster Metadata
+Before going further, answer the following questions about our `RasterStack`.
 
-#Plotting Time Series Data
+1. What is the `CRS`?
+2. What is the `resolution` of the data? And what `units` is that resolution in?
+
+``` {r challenge-code-raster-metadata, eval=FALSE, echo=FALSE}
+#1. UTM zone 19 WGS 84
+#2. 30x30 meters
 
-Once we have created our `RasterStack`, we can visualize our data. Remember 
-from a previous lesson, that we can use the `plot` command to quickly plot a `RasterStack`.
+```
 
+#Plotting Time Series Data
+Once we have created our `RasterStack`, we can visualize our data. We can us
+the `plot()` command to quickly plot a `RasterStack`.
 
 ```{r plot-time-series }
 
-#view a histogram of all of the rasters
-#nc specifies number of columns
+#view a plot of all of the rasters
+#'nc' specifies number of columns (we will have 13 plots)
 plot(NDVI_stack, 
      zlim = c(1500, 10000), 
      nc = 4)
 
 ```
 
```

Comment:
Wait, the accepted values for NDVI range from 0-1. Why does our data range from 0 - 10,000? The metadata for this NDVI data specifies a scale factor: 10,000. A scale factor is
sometimes used to remove decimals. Storing decimal places creates larger file sizes compared to storing
integer values.